handlers/kinetic_request_ce_submission_db_insert_v1/handler/init.rb

```diff
@@ -160,7 +159,7 @@ def initialize(input)
     
     # Output SQL statements if the 'trace' level info parameter is set to true.
     @db.sql_log_level = :debug if @enable_trace_logging
-    @db.logger = Logger.new($stdout) if @enable_debug_logging
+    @db.logger = org.apache.log4j.LogManager.getLogger("com.kineticdata") if @enable_trace_logging
```

Comment:
Do we need to check if running on Task 4 and adjust the log4j package name? I seem to remember the package name changed between log4j 1.2 (task 4) and log4j 2.0 (task 5).

Reply:
After testing `org.apache.log4j.LogManager.getLogger("com.kineticdata")` in Task v5.2.8 and v4.5.0-SNAPSHOT it appears that the line of code is supported across many versions of Task.  Should there still be a change made?